lark/grammars/python.lark

```diff
@@ -111,7 +114,8 @@ suite: simple_stmt | _NEWLINE _INDENT stmt+ _DEDENT
 ?comparison: expr (comp_op expr)*
 star_expr: "*" expr
 
-?expr: or_expr
+?expr: or_expr | assign_expr
+assign_expr: NAME ":=" expr
```

Comment:
Chaining assign_expr isn't allowed.

Reply:
I think assign_expr should have `test` on the right side, and I'm not sure if it belongs in `expr` itself
```suggestion
assign_expr: NAME ":=" test
```

Reply:
Yes, this is true. No, it shouldn't be inside of expression, it needs to be on a higher level than the logical operators and comparison.